cmd/wego/add/cmd.go

```diff
@@ -54,26 +57,38 @@ func runCmd(cmd *cobra.Command, args []string) {
 
 	authMethod, err := ssh.NewPublicKeysFromFile("git", params.PrivateKey, params.PrivateKeyPass)
 	if err != nil {
-		fmt.Fprintf(shims.Stderr(), "failed reading ssh keys: %s\n", err)
-		shims.Exit(1)
+		fmt.Fprintf(os.Stderr, "failed reading ssh keys: %s\n", err)
+		os.Exit(1)
 	}
 
+	if params.Url == "" {
+		if len(args) == 0 {
+			fmt.Fprint(os.Stderr, "no app --url or app location specified")
```

Comment:
I've noticed with most wego commands (including subcommands), that leaving off parameters/etc at the end spits out an error message with usage output, but not `wego app add`.  For example, the command `wego app status` gives this output:

```
$ wego app status
Error: requires at least 1 arg(s), only received 0
Usage:
  wego app status <app-name> [flags]

Examples:
wego app status podinfo

Flags:
  -h, --help   help for status

Global Flags:
      --namespace string   gitops runtime namespace (default "wego-system")
  -v, --verbose            Enable verbose output

Error: requires at least 1 arg(s), only received 0
```
Should we have `wego app add` follow this convention as well?

Reply:
good point, I'll check out how to show that.